src/views/preview/preview.jsx

```diff
@@ -329,6 +329,7 @@ class Preview extends React.Component {
                 </Page> :
                 <IntlGUI
                     enableCommunity
+                    hideIntro
```

Comment:
For correctness should also add this to the other instance of GUI in `presentation.jsx`. It won’t matter now but could if we reorganize modals in the future.